src/App.tsx

```diff
@@ -1,14 +1,58 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selected, setSelected] = useState<Todo | null>();
+  const [filterSelect, setFilterSelect] = useState('');
+  const [search, setSearch] = useState('');
+
+  const getTodosFromServer = async () => {
+    let todosArr;
+
+    try {
+      todosArr = await getTodos();
+    } catch (error) {
+      throw new Error('Error todos loading');
```

Comment:
In case of exception, the error is being thrown, but it is not being caught anywhere. It would be better to handle that error in a way that does not crash the entire application.